index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Taher Haggui </title>
+        <link rel="stylesheet" type="text/css" href="styles/style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav>
+            <a class="logo" href="#">My Logo</a>
+            
+            <img src="statics/images/Union.png" alt="Menu">
+               
+        </nav>
```

Comment:
- [ ] This section is supposed to be wrapped inside a `header` tag in order to enhance HTML semantics. Please ensure you do so. 
- [ ] Please ensure that you follow the Figma design in assigning the margins as well as positioning the different elements within the page.